.github/workflows/setup_pwa_manual.yml

```diff
@@ -0,0 +1,137 @@
+name: SalesforceCommerceCloud/pwa-kit/e2e
+on:
+  workflow_dispatch:
+    inputs:
+      use_extensibility:
+        type: boolean
+        description: Use Extensibility?
+        default: true
+      project_name:
+        type: string
+        description: Project Name
+        default: "scaffold-pwa"
+      instance_url:
+        type: string
+        description: Instance Url
+        default: "https://zzrf-002.dx.commercecloud.salesforce.com"
```

Comment:
Why `zzrf-002` and not `zzrf-001`?

Reply:
no specific reason, I was just testing with `zzrf-002`. We can change to `zzrf-001`

Reply:
Were you planning to make this change still @unandyala , does seem preferable to use `zzrf-001`